src/Auth0.php

```diff
@@ -412,6 +437,23 @@ public function setRefreshToken(
         return $this;
     }
 
+    /**
+     * Sets and persists the refresh token.
+     *
+     * @param string $refreshToken - refresh token returned from the code exchange.
+     */
+    public function setTokenExpiration(
+        string $tokenExpiration
+    ): self {
+        $this->state->setTokenExpiration($tokenExpiration);
+
+        if ($this->configuration->hasSessionStorage() && $this->configuration->getPersistAccessToken()) {
+            $this->configuration->getSessionStorage()->set('tokenExpiration', $tokenExpiration);
```

Comment:
Can we call it `accessTokenExpiration`? Expiration of the other tokens can be a source of confusion

Reply:
Yup, good callout there. I've updated the naming for clarity 👍